### PR TITLE
Switched logger implementation to zap

### PR DESCRIPTION
**Pull request by gustafj**

### CODE REVIEW

File: log/log.go [hunk 1]
```diff
@@ -36,42 +37,26 @@ type Logger interface {
...
 func init() {
-	SetFormatter(&logrus.JSONFormatter{})
-	CallTrace(false)
-	Source(true)
-}
-
-func Base() Logger {
-	return baseLogger
-}
+	l, err := zap.NewProduction()
```
Comment (stroem) [inline log/log.go]: Should we expose `l.Sync()`?

Comment (stroem) [inline log/log.go]: Use timestamp in RFC3339?

Comment (gustafj) [reply log/log.go]: I think so, fixed in new patch

Comment (gustafj) [reply log/log.go]: Agree, fixed in new patch

**Pull request merged**